model/riscv_sys_regs.sail

```diff
@@ -350,6 +347,8 @@ function in32BitMode() -> bool = {
 /* F and D extensions have to enabled both via misa.{FD} as well as mstatus.FS */
 function haveFExt()    -> bool = (misa.F() == 0b1) & (mstatus.FS() != 0b00)
 function haveDExt()    -> bool = (misa.D() == 0b1) & (mstatus.FS() != 0b00)
+/* Zfh (half-precision) extension */
+function haveZfh()     -> bool = mstatus.FS() != 0b00
```

Comment:
This also needs `(misa.F() == 0b1)`; Zfh depends on F.

Reply:
Done, sorry for the delay.